README.md

```diff
@@ -37,3 +39,24 @@ Lesson plan:
 
 [11 Mapping Marc to Dublin Core](./docs/11_MARC_to_Dublin_Core.html)
 
+## Testing
+
+### Installation
+If you are on debian derivates go install build-essentials:
+```
+apt install build-essential
+```
+Then you can use the Ruby Dependency Management to build all you need:
+```
+bundle install
```

Comment:
After running `sudo apt install build-essential` and then `bundle install`, I get:

> $ bundle install
> Command 'bundle' not found, but can be installed with:
> sudo apt install ruby-bundler

Is it only me or should installatioj of `ruby-bundler` also be added to the documentation?

However, after installing `ruby-bundler`, `bundle install` will start but throw errors for me. So, up until now I did not manage to start jekyll...

Reply:
@acka47 could you try this step-by-step (as root or sudoing and beeing in this repo's directory):
```
apt remove ruby-bundler; apt purge jekyll; apt update; apt install build-essential; bundle install; bundle exec jekyll serve
```
Ignore Warnings , the server should start at http://127.0.0.1:4000. If it does not , I am clueless.